tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -27,15 +27,26 @@ WORKDIR /openedx/discovery
 RUN echo "{}" > /openedx/config.yml
 ENV DISCOVERY_CFG /openedx/config.yml
 
-# Install python venv
-RUN python3 -m venv ../venv/
+# Install pyenv
```

Comment:
Can we inherit FROM the python:3.12-bookworm image instead of installing python manually with pyenv? These steps make the build more complex, and I'd rather not use them if possible at all.

Reply:
P.S. this is on my todo list to test out python build using python images. That can be done in a followup PR, in my opinion (once tested).
Why do we want to use bookworm specifically? why not slim?